samples/swift/uidemo/AppDelegate.swift

```diff
@@ -32,5 +33,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     FIRApp.configure()
     return true
   }
+  
+  func application(app: UIApplication, openURL url: NSURL, options: [String : AnyObject]) -> Bool {
+    let sourceApplication = options[UIApplicationOpenURLOptionsSourceApplicationKey] as! String?
+    // Seems like an oversight that this API doesn't take a nullable sourceApplication.
+    if FIRAuthUI.authUI()?.handleOpenURL(url, sourceApplication: sourceApplication!) ?? false {
```

Comment:
`sourceApplication` is String optional. So, I think that `sourceApplication!` could be nil and make a runtime errors. Isn't it?

how was this?
```swift
    if let sourceApplication = options[UIApplicationOpenURLOptionsSourceApplicationKey] as! String? {
        // Seems like an oversight that this API doesn't take a nullable sourceApplication.
        if FIRAuthUI.authUI()?.handleOpenURL(url, sourceApplication: sourceApplication) ?? false {
```